src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -277,7 +279,7 @@ private String getPendingOrders() throws Exception {
 			}
 		} catch (FileNotFoundException e) {
 			String msg = e.getMessage().replace(getConfig().getString("key"), "privateKey");
-			WooMinecraft.instance.wmc_log(msg);
+			throw new FileNotFoundException(msg);

```

Comment:
instead of throwing an exception, just `return ""` an empty string here. Since it's caught in the `pendingOrders.isEmpty()` check inside the `check()` method.